runtime/src/accounts_db.rs

```diff
@@ -634,6 +642,18 @@ impl AccountsDB {
     // can be purged because there are no live append vecs in the ancestors
     pub fn purge_zero_lamport_accounts(&self, ancestors: &HashMap<u64, usize>) {
         self.report_store_stats();
+
+        let last_zla_count = self.last_purge_zla_count.load(Ordering::Relaxed);
+        let zla_count = self.zla_count.load(Ordering::Relaxed);
+        let prev_zla_count = self.last_purge_zla_count.compare_and_swap(
```

Comment:
I think this could be replaced with this?: https://doc.rust-lang.org/std/sync/atomic/struct.AtomicUsize.html#method.compare_exchange